### PR TITLE
feat(.env): bump helm-docs version

### PR DESCRIPTION
**Pull request by eyenx**
bump version of helm-docs to v1.4.0

### CODE REVIEW

**Review by michaelimfeld — DISMISSED** [review]

**Comment by eyenx** [comment]
@michaelimfeld pls review again, checks should be successful now

File: charts/timed/Chart.yaml [hunk 1]
```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: timed
 description: Chart for Timed application
 type: application
-version: 0.4.10
-appVersion: v1.1.2
+version: 0.4.11
+appVersion: v1.1.3
```
Comment (michaelimfeld) [inline charts/timed/Chart.yaml:6]: Why did you bump the appVersion?

Comment (eyenx) [reply charts/timed/Chart.yaml:6]: because all app needed a version bump after changing the dependency of helm-docs (generating new README.md)

Comment (eyenx) [reply charts/timed/Chart.yaml]: oh my bad. AppVersion does not need to be bumped... let me fix that

Comment (eyenx) [reply charts/timed/Chart.yaml]: 6f63f9d fixed this

Comment (michaelimfeld) [reply charts/timed/Chart.yaml]: Chart version yes...

**Review by michaelimfeld — APPROVED** [review]

**Comment by hairmare** [comment]
Should we consider removing the version of helm-docs from the end of all readmes? It feels wrong bumping all the charts if the only change to the README is the version of the tool esp. given the release cadence of helm-docs. Maybe like this:

----------------------------------------------
Autogenerated from chart metadata using [helm-docs](https://github.com/norwoodj/helm-docs)

**Comment by eyenx** [comment]
yes

**Pull request merged**